daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -320,31 +320,68 @@ object Runner {
   ): Future[SValue] = {
     val darMap = dar.all.toMap
     val compiledPackages = PureCompiledPackages.assertBuild(darMap, Runner.compilerConfig)
+    def converter(json: JsValue, `type`: Type) = {
```

Comment:
```suggestion
    def converter(json: JsValue, typ: Type) = {
```
I prefer not to use the back-quote feature, except maybe in pattern matching.